openwrt-openwisp-monitoring/files/monitoring.init

```diff
@@ -52,8 +52,9 @@ start_service() {
     monitored_interfaces="--monitored_interfaces $monitored_interfaces"
 
     procd_open_instance "openwisp_monitoring_monitoring"
-    procd_set_param command $PROG "$base_url" "$uuid" "$key" "$verify_ssl" "$interval" "$monitored_interfaces"
+    procd_set_param command $PROG $base_url $uuid $key $verify_ssl $interval $monitored_interfaces
     procd_set_param respawn "${respawn_threshold:-3600}" "${respawn_timeout:-5}" "${respawn_retry:-5}"
+    procd_set_param stderr 1
```

Comment:
Adding this to log all errors in the instance so that it can be accessed using logread